apps/web/components/Embed.tsx

```diff
@@ -1058,7 +1058,7 @@ const EmailEmbed = ({ eventType, username }: { eventType?: EventType; username:
     shallow
   );
   const event = useEvent();
-  const schedule = useScheduleForEvent();
+  const schedule = useScheduleForEvent({ eventId: eventType?.id });
```

Comment:
Not passing the org here?

Reply:
yeah because we are passing eventId directly here

<img width="935" alt="Screenshot 2023-08-08 at 6 19 54 PM" src="https://github.com/calcom/cal.com/assets/53316345/b1e25d81-f96c-4219-9fa8-0bdd9ef5674e">